json_serializable/lib/src/utils.dart

```diff
@@ -11,6 +11,42 @@ import 'package:analyzer/src/dart/resolver/inheritance_manager.dart'
 
 import 'package:source_gen/source_gen.dart';
 
+// TODO: still need handle tripe singe/double quotes within `value`
+String escapeDartString(String value) {
+  if (value.contains('\n')) {
+    return "r'''\n$value'''";
```

Comment:
why the leading `/n`?

Reply:
Because one leading `\n` is ignored. So if it's the first character in the string, it's skipped.

Reply:
(Also ignores leading white space if that's all there is before the first line terminator).